### ISSUE TITLE
New issue with new release (0.0.5)

### ISSUE THREAD

**Issue by multicam** [mrcljx/grunt-liquid#11]
Hi there,

After updating to the new release, this error appeared, it might be just a typo -- hopefully ;)

Thanks for looking into this.
Cheers

```
Loading "liquid.js" tasks...ERROR
>> TypeError: Object function Template() {
>>       this.registers = {};
>>       this.assigns = {};
>>       this.instanceAssigns = {};
>>       this.tags = {};
>>       this.errors = [];
>>       this.rethrowErrors = true;
>>     } has no method 'registerTag'
Warning: Task "liquid:pages" not found. Use --force to continue.
```

**Comment by multicam** [comment]
if that can help, I noticed that the file "tasks/lib/liquid-ext.coffee" has not been converted to js -- this is where the actual `registerTag` function is declared.
Please can you provide an ETA -- it's blocking me right now, but it looks like a minor issue to fix

**Comment by mrcljx** [comment]
This should be fixed by 0.0.6 - could you confirm this?

**Comment by mrcljx** [comment]
Correction - should be fixed in 0.0.7. ;)
